.github/workflows/ci.yaml

```
@@ -0,0 +1,305 @@
+---
+name: ci
+
+"on":
+  push:
+    branches:
+      - 'main'
```

🔧 Looks like this repo is using `master`
```suggestion
      - 'master'
```